modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/parcellationmeshgeneration/v0.1.0.json

```diff
@@ -18,7 +18,7 @@
       "targetClass": "nsg:ParcellationMeshGeneration",
       "and": [
         {
-          "@id": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0/shapes/ActivityShape"
+          "node": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0/shapes/ActivityShape"
```

Comment:
bumped "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0" to "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.4"